Command/RemoveThumbsCommand.php

```diff
@@ -238,4 +234,9 @@ protected function log($message)
             $this->output->writeln($message);
         }
     }
+
+    private function getQuestionHelper(): QuestionHelper
```

Comment:
This won't work with PHP < 7.0

Reply:
I see that php < 7.1 is dropped in 3.x, but if I have php 7.0 and sf3, how can I receive this patch?

Reply:
@ZRazor by upgrading, or by ignoring platform requirements with composer (not recommended)

Reply:
@ZRazor we ended up restoring compatibility.
@vyshkant that means you need to change that.

Reply:
Well it's actually not done yet, see https://github.com/sonata-project/SonataMediaBundle/pull/1314